<ui version="4.0" >
 <class>ManageFeatureCollectionsDialog</class>
 <widget class="QDialog" name="ManageFeatureCollectionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Manage Feature Collections</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../qt-resources/qt_widgets.qrc" >:/gnome_preferences_desktop_16.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Manage the feature collections which are loaded in the application.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_feature_collections" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode" >
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="gridStyle" >
      <enum>Qt::DashLine</enum>
     </property>
     <column>
      <property name="text" >
       <string>File Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>File Format</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>In Use?</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Actions</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="button_open_file" >
       <property name="text" >
        <string>&amp;Open File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save_all" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Save All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_close" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_close</tabstop>
  <tabstop>button_open_file</tabstop>
  <tabstop>button_save_all</tabstop>
  <tabstop>table_feature_collections</tabstop>
 </tabstops>
 <resources>
  <include location="../qt-resources/qt_widgets.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>ManageFeatureCollectionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>595</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel" >
     <x>596</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
